build/gulp/tasks/perf.ts

```diff
@@ -72,6 +73,53 @@ const normalizeMeasures = (measures: ProfilerMeasureCycle[]) => {
   }))
 }
 
+const getPercentDiff = (minValue: number, actualValue: number): number =>
+  _.round((actualValue / minValue) * 100 - 100, 2)
+
+const createMarkdownTable = (
+  data,
```

Comment:
can we introduce type for `data` arg?

Reply:
renamed to `normalizedMeasures` and add more types 👍